__tests__/workers/notifications.ts

```diff
@@ -1734,4 +1733,36 @@ it('should add squad reply notification', async () => {
     '4',
   );
   expect(actual[0].ctx.userIds).toIncludeSameMembers(['1', '3', '2']);
+
+  // add new comment for testing notification after first comment
+  // should notify users who commented on the first comment
+  // not userId 3 who is the one who is writing
+  await con.getRepository(Comment).save([
+    {
+      id: 'c6',
+      postId: 'p1',
+      userId: '3',
+      content: 'sub comment5',
+      createdAt: new Date(2020, 1, 6, 0, 0),
+      parentId: 'c1',
+    },
+  ]);
```

Comment:
If you can add additional test case for this and not modify the current test. So something in line `author should get a reply notification if they commented`, same for scout (it can be two test cases).

Reply:
@capJavert I've updated with a more generic one `users should get a reply notification if they commented in the same thread`, is this ok? Because the fix is involving the thread users more than the author/scout. What do you think?

Reply:
yeah, looks good, let's keep are eyes also on any reports with this after merge if we missed some edge case 🙏